eks-distro-base/scripts/install_binary

```diff
@@ -70,6 +70,7 @@ function build::install::binary() {
     # we need to remove the link before copying
     [ -L $final_bin ] && unlink $final_bin
 
+    mkdir -p $(dirname $final_bin)
```

Comment:
surprised we hadnt seen this before, but this just ensures the dest folder exists before trying to copy.